packages/api/src/services/actions.ts

```diff
@@ -140,6 +140,15 @@ export function Handlers(config: Config, appState: Dependencies): Actions {
     async listTvms(currency: CurrencySymbol = "usd") {
       return appState.stats[currency].latest.tvm.values();
     },
+    async historicalMarketPricesBetween(tokenAddress: string, start = 0, end: number = nowS()) {
+      assert(tokenAddress, "requires token address");
+      return marketPrices.usdc.history.betweenByAddress(tokenAddress, start, end);
+    },
+    async sliceHistoricalMarketPrices(tokenAddress: string, start = 0, length = 1) {
+      assert(tokenAddress, "requires token address");
+      assert(length < 1000, "length must be less than 1000 samples");
```

Comment:
Why this magic number?

Reply:
good question, i think chris brought up a while ago we should probably limit how many samples get returned from api.  This caps it at 1000. should probably change this into a config var though!

Reply:
going to save this change for a chore ticket, its the last task here https://app.clubhouse.io/uma-project/story/1636/various-api-improvements